zebra-rpc/Cargo.toml

```
@@ -21,6 +24,7 @@ jsonrpc-derive = "18.0.0"
 jsonrpc-http-server = "18.0.0"
 
 tokio = { version = "1.16.1", features = ["time", "rt-multi-thread", "macros", "tracing"] }
+tower = { version = "0.4.12", features = ["hedge", "limit"] }
```

Optional: this will build slightly faster, because it disables unused features

```suggestion
tower = "0.4.12"
```

---

Thanks,in a hurry it seems this was copied&pasted from another Cargo.toml file. I removed those additional features at https://github.com/ZcashFoundation/zebra/pull/3707/commits/f2810ba8fd0a9e4346515a17eeee2b7ca1b0102b